views/author.handlebars

```diff
@@ -5,52 +5,65 @@
   <title>Autor</title>
   <link rel="stylesheet" href="/css/author.css">
   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Fjalla+One">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.6.0/css/bulma.min.css">
+
 </head>
 <body>
-  <section class="author-section">
-    <header class="author-header">
-      <div class="logo">
-        <h1><a href="/" class="link-logo">&#60;/CONHECIMENTO LIVRE&#62;</a></h1>
+  <nav class="navbar is-primary" role="navigation" aria-label="main navigation" style=" z-index: 3;">
```

Comment:
Inline Style não é recomendado.

Ref: https://stackoverflow.com/questions/2612483/whats-so-bad-about-in-line-css